src/fontra/views/editor/editor.js

```diff
@@ -2030,6 +2104,259 @@ export class EditorController {
     return { contentElement, warningElement };
   }
 
+  selectionHasLockedGuidelines() {
+    const {
+      guidelineGlyph: guidelineGlyphSelection,
```

Comment:
Same remark here about the name. Let's call this "guideline", and use "fontGuideline" later, once we've figured out how to deal with them.

Reply:
I renamed all guidelines.